### PR TITLE
fix(Core/Creature): Move hardcoded text/gossips to DB and minor fixes - boss_freya

### PR DESCRIPTION
**Pull request by avarishd**
Part of https://github.com/azerothcore/azerothcore-wotlk/projects/15 Going trough the entirety of Ulduar
creature_text for Freya is already populated in the DB 
## Changes Proposed:
-  Added the following missing emotes:
- *EMOTE_ALLIES_OF_NATURE
- *EMOTE_LIFEBINDERS_GIFT
- *EMOTE_GROUND_TREMOR
- *EMOTE_IRON_ROOTS
- Alive Elders no longer talk if starting Freya hardmode.
- Fixed Freya instantly despawning after being defeated.
- Completing hardmode now despawn any alive Elders, instead of killing them. (This also fixes elders talking as if they've killed someone, themselves in that case, when killed by the previous code, `UNIT::Kill`)

## Issues Addressed:

- Closes none

## SOURCE:

[YT](https://www.youtube.com/results?search_query=freya+wow)
## Tests Performed:

- Tested in-game with sound

## How to Test the Changes:

1. .go c 136554
2. Interact with the Elders (kill/leave alive).
3. Pull (with sound enabled) and make sure none of the alive elders is speaking if engaging hardmode.
4. Verify that all of the previously missing emotes are there.
5. After defeating Freya hardmode pay attention to the elders, none should be speaking and they should despawn instead of being killed.
6. Freya should stay for couple (5) seconds after being successfully defeated, after that should despawn.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Kitzunu — APPROVED** [review]

**Comment by mpfans** [comment]
`Completing hardmode now despawn any alive Elders, instead of killing them. (This also fixes elders talking as if they've killed someone, themselves in that case, when killed by the previous code, UNIT::Kill)`

In `The Obsidian Sanctum`, the same bug exists when you directly kill `Sartharion` without killing three dragons

**Comment by Nyeriah** [comment]
@avarishd Please don't include functionality changes in the same PRs as the text/gossip refactors. Or specify what is being changed in the commit message instead of the text/gossip updates, so we don't end up accidentally merging things that would require a better accessment and perhaps testing

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp [hunk 1]
```diff
@@ -360,7 +337,7 @@ class boss_freya : public CreatureScript
                         continue;
 
                     if (Creature* e = ObjectAccessor::GetCreature(*me, _elderGUID[i]))
-                        Unit::Kill(e, e);
+                        e->AI()->DoAction(ACTION_ELDER_FREYA_KILLED);
```
Comment (Nyeriah) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:340]:
```cpp
e->DespawnOrUnsummon();
```

Comment (avarishd) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:340]: @Nyeriah Sorry, didn't know. And `e->DespawnOrUnsummon();`  is better solution,

**Pull request merged**